### PR TITLE
[MIG] website_sale_attribute_filter_form_submit: Migration to 12.0

### PR DESCRIPTION
**Pull request by chienandalu**
Backport of #487 with bugfixes and improvements like:

- Visual feedback when a color is changed
- Enable/Disable at website level
- Proper handling of behavior fallbacks
manual_filter_configuration

manual_filter_usage

cc @Tecnativa TT29431

ping @Yajo @Tardo

### CODE REVIEW

File: website_sale_attribute_filter_form_submit/README.rst [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+.. |badge4| image:: https://img.shields.io/badge/weblate-Translate%20me-F47D42.png
+    :target: https://translation.odoo-community.org/projects/e-commerce-12-0/e-commerce-12-0-website_sale_attribute_filter_form_submit
+    :alt: Translate me on Weblate
+.. |badge5| image:: https://img.shields.io/badge/runbot-Try%20me-875A7B.png
+    :target: https://runbot.odoo-community.org/runbot/113/12.0
+    :alt: Try me on Runbot
+
+|badge1| |badge2| |badge3| |badge4| |badge5| 
+
+This module hides all the submit elements of the attribute form filter to block
```
Comment (pedrobaeza) [inline website_sale_attribute_filter_form_submit/README.rst]: This is a very technical explanation. The readme should be for users...

File: website_sale_attribute_filter_form_submit/README.rst [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+Do not contact contributors directly about support or help with technical issues.
+
+Credits
+=======
+
+Authors
+~~~~~~~
+
+* Studio73
+* Tecnativa
```
Comment (pedrobaeza) [inline website_sale_attribute_filter_form_submit/README.rst:77]: Should we share co-authorship?

File: website_sale_attribute_filter_form_submit/__manifest__.py [hunk 3]
```diff
@@ -0,0 +1,18 @@
+# Copyright 2021 Studio73 - Miguel Gandia
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+{
+    "name": "Website Sale Attribute filter form submit",
```
Comment (pedrobaeza) [inline website_sale_attribute_filter_form_submit/__manifest__.py]: Put better name

File: website_sale_attribute_filter_form_submit/README.rst [hunk 4]
```diff
@@ -0,0 +1,81 @@
...
+|badge1| |badge2| |badge3| |badge4| |badge5| 
+
+This module hides all the submit elements of the attribute form filter to block
+submitting it automatically in the 'onchange' of the attributes values and adds
+a submit button so the user applies them manually.
+
+**Table of contents**
+
+.. contents::
+   :local:
```
Comment (pedrobaeza) [inline website_sale_attribute_filter_form_submit/README.rst:35]: Add usage

**Review by pedrobaeza — CHANGES_REQUESTED** [review]
Trying on runbot I don't get the Submit button and the filters are applied automatically.

File: website_sale_attribute_filter_form_submit/README.rst (hunk 2 above)
Comment (chienandalu) [reply website_sale_attribute_filter_form_submit/README.rst:77]: The truth is that the original implementation wasn't fully operative, so we made a good amount of changes

**Comment by chienandalu** [comment]
Changes done. Also added configuration section so it's clear how to set it up

**Review by pedrobaeza — APPROVED** [review]
Functional review on runbot

File: website_sale_attribute_filter_form_submit/templates/website_sale.xml [hunk 5]
```diff
@@ -0,0 +1,34 @@
...
+    </template>
+    <template
+        id="products_attributes_filter_manual"
+        name="Manual filters"
+        inherit_id="website_sale.products_attributes"
+        active="False"
+        customize_show="True" 
+    >
+        <xpath expr="//form[hasclass('js_attributes')]" position="inside">
+            <div class='pt-2 pb-2'>
```
Comment (Tardo) [inline website_sale_attribute_filter_form_submit/templates/website_sale.xml]:
```suggestion
            <div class='py-2'>
```

**Review by Tardo — APPROVED** [review]
LGTM :+1: Tested on runbot. Only non-blocking comment.

File: website_sale_attribute_filter_form_submit/templates/website_sale.xml (hunk 5 above)
Comment (chienandalu) [reply website_sale_attribute_filter_form_submit/templates/website_sale.xml]: Nice. Good to know the abbreviated way

**Pull request merged**